src/math/Plane.js

```diff
@@ -122,6 +122,47 @@ THREE.Plane.prototype = {
 
 	},
 
+	intersectRay: function(ray, optionalTarget){
+
+		var result = optionalTarget || new THREE.Vector3();
+
+		var plane = {
+			origin: this.coplanarPoint(),
+			direction: this.normal
+		}
```

Comment:
It seems unusual to instantiate an alternate "plane" structure inside a `Plane` method. You probably do not need this construct.